pkg/sources/github/README.md

```diff
@@ -0,0 +1,99 @@
+Knative GitHub Source 
+=====================
+
+This source will enable receiving of GitHub events from within Knative Eventing.
+
+## Events
+
+- dev.knative.github.pullrequest
+
+## Images
+
+Uses two images, the feedlet and the receive adapter.
+
+### Feedlet
+
+A Feedlet is a container to create or destroy event source bindings.
+
+The Feedlet is run as a Job by the Feed controller.
+
+This Feedlet will create the Receive Adapter and register a webhook in GitHub
+for the account provided.
+
+### Receive Adapter
+
+The Receive Adapter is a Service.serving.knative.dev resource that is registered
+to receive GitHub webhook requests. The source will wrap this request in a
+CloudEvent and send it to the action provided. 
+
+## Usage
+
+The GitHub Source expects there to be a secret in the following form:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: githubsecret
+type: Opaque
+stringData:
+  githubCredentials: >
+    {
+      "accessToken": "<YOUR PERSONAL TOKEN FROM GITHUB>",
+      "secretToken": "<YOUR RANDOM STRING>"
+    }
+
+```
+
+The Feedlet requires a ServiceAccount to run as a cluster admin for the targeted namespace: 
```

Comment:
It's unfortunate that we can't put this in a yaml file somewhere to happen at the same time as the controller is installed/the source is created.